src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -2018,105 +2229,6 @@ public void speakBlocking(String format, Object... args) {
     }
   }
 
-  @Deprecated /* use startPeers */
-  public void startAll() throws Exception {
-    startAll(null, null);
-  }
-
-  @Deprecated /* use startPeers */
-  public void startAll(String leftPort, String rightPort) throws Exception {
```

Comment:
similar to above, can we just have this call startPeers?  leave this method in for convenience sake?